### ISSUE TITLE
Nested capture is not aware of bygroups(None)

### ISSUE THREAD

**Issue by thatch** [thatch/regexlint#7]
False positive as of pygments-main rev a54d5fcaae14

```
(r'(cycle)(\s+)(([^\s:]*)(:))?(\s*)',
 bygroups(Text, Text, None, using(this), Text, Text),
```
